src/site/content/en/metrics/lcp/index.md

```diff
@@ -381,6 +381,17 @@ performance techniques that can also improve LCP, see:
 * [Optimize your JavaScript](/fast#optimize-your-javascript) (for
   client-rendered sites)
 
+### When and when not to lazy load images
+
+Lazy loading is a common performance optimization in which images and other elements are deferred from loading until they approach the viewport as the user scrolls. It's a time-tested technique that saves bandwidth and improves load performance.
+
+However, lazy loading can affect LCP negatively when used improperly. As a general rule, these is one rule of lazy loading to follow if your LCP candidate is an image: If an LCP candidate is above the fold (or reasonably close), do _not_ lazy load it. The reason for this depends on how you're lazy loading images:
```

Comment:
Saying "rule" twice here reads a bit awkwardly. How about "In general, there is one rule..."?

Reply:
```suggestion
However, lazy loading can affect LCP negatively when used improperly. In general, there is one rule of lazy loading to follow if your LCP candidate is an image: If an LCP candidate is above the fold (or reasonably close), do _not_ lazy load it. The reason for this depends on how you're lazy loading images:
```